go/base/static/css/conversations.less

```diff
@@ -1,3 +1,6 @@
+@import "./lesshat";
```

Comment:
It helps by making my editor stop complaining about unrecognised mixins :) is it pulling in the stylesheet more than once as a result?